lark/parsers/earley.py

```diff
@@ -296,13 +297,13 @@ def parse(self, lexer, start):
                 debug_walker.visit(solutions[0], "sppf.png")
 
 
-        if len(solutions) > 1:
-            assert False, 'Earley should not generate multiple start symbol items!'
-
         if self.Tree is not None:
             # Perform our SPPF -> AST conversion
             transformer = ForestToParseTree(self.Tree, self.callbacks, self.forest_sum_visitor and self.forest_sum_visitor(), self.resolve_ambiguity)
-            return transformer.transform(solutions[0])
+            solutions = [transformer.transform(s) for s in solutions]
+
+            return solutions[0] if len(solutions) == 1 else self.Tree('_ambig', solutions)
```

Comment:
For multiple solutions, it is possible for one or more of the solutions to be `_ambig` nodes themselves, in which case their children could be inlined as children of the `_ambig` node being created here.

This is probably just a matter of tree aesthetics.

Reply:
Done.